.github/workflows/test-download-artifact-s3-action.yml

```diff
@@ -0,0 +1,119 @@
+name: Test
```

Comment:
Should give this workflow a more specific name as this repo has a lot of workflows.

Reply:
Added, sorry I must have missed this